docs/features/scripts.md

```diff
@@ -0,0 +1,50 @@
+---
+id: scripts
+title: Scripts support
+---
+
+### Working with a script
+
+Whenever, Metals opens a script file with `*.sc` extension, but only when it's
+not `*.worksheet.sc` or a build server is not already started for it, users will
+be prompted to choose whether they want to use [Ammonite](http://ammonite.io/)
+or [ScalaCli](https://scala-cli.virtuslab.org/) for using within the it.
```

Comment:
```suggestion
or [ScalaCli](https://scala-cli.virtuslab.org/) for usage within it.
```

Reply:
I actually think this is worded a bit oddly. I'l recommend actually changing the whole ending to something like:

> for it.

If you want to keep it really simple, but if you want to be more detailed something like:

> the power the script.